types/auth0/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for auth0 3.0
+// Type definitions for auth0 4.0
```

Comment:
This should be 2.3.0. I accidentally changed it to 3.0 but it should be according to auth0 version